howso/typing.amlg

```diff
@@ -1117,6 +1117,20 @@
                         selected_prediction_stats {ref "SelectedPredictionStats"}
                     )
             )
+        AblationThresholdMap
+            (assoc
+                type "assoc"
+                description
+                    (concat
+                        "Map of prediction stats to map of feature name to threshold value. Thresholds can be absolute, delta, or relative depending "
+                        "on which parameter is being used."
+                    )
+                values
+                    (assoc
+                        type "assoc"
+                        values (assoc type "number")
```

Comment:
we updated the typing to not support "values" for assocs, gotta use 'additional_indices' now for unnamed indices